src/styles/components/card.scss

```diff
@@ -0,0 +1,89 @@
+.product-card {
+  width: $card-width;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-top: 46px;
+  }
+
+  &__image {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    @include font(500, 12px, 18px, $main-accent-color);
+    padding-top: 40px;
+  }
+
+  &__code {
+    @include font(400, 10px, 14px, $secondary-color);
+    padding-top: 4px;
+  }
+
+  &__rating-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    padding-top: 16px;
+  }
+
+  &__reviews {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $main-accent-color;
+  }
+
+  &__price-container {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    padding-top: 24px;
+  }
+
+  &__text {
+    @include font(400, 12px, 18px, $secondary-color);
+  }
+
+  &__price {
+    @include font(700, 16px, 18px, $main-accent-color);
+  }
+
+  &__button {
+    display: block;
+    width: 166px;
```

Comment:
you don't need width here, it's redundant property

Reply:
Fixed this and other mistakes. Thank you for your review ;)